### PR TITLE
ROE-411 update corporate manager officer page to V5

### PR DESCRIPTION
**Pull request by SVarallo**
### JIRA link
https://companieshouse.atlassian.net/browse/ROE-168

### Change description
Updated corporate manger officer html to match the latest in V5 of the prototype as well as updating the managing officer corporate data model to the latest specified in the spec

### Work checklist

- [x] Tests added where applicable
- [x] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Review by mwestacott — APPROVED** [review]
Looks good to me. Suggest getting a second approval before merging though.

**Review by mouhajer-ch — COMMENTED** [review]
As @markpit suggested in the chat we need to update [here](https://github.com/companieshouse/overseas-entities-web/blob/68a4a2f42183c43498e8a9f24accb8d1fb35ae10/views/beneficial-owner-type.html#L74)  and [here](https://github.com/companieshouse/overseas-entities-web/blob/main/views/includes/list/managing-officer.html#L1)

**Review by mouhajer-ch — APPROVED** [review]
Thanks @SVarallo for the bug fix

**Pull request merged**
